lib/internal/child_process.js

```diff
@@ -456,16 +456,22 @@ function setupChannel(target, channel) {
       }
       chunks[0] = jsonBuffer + chunks[0];
 
+      var nextTick = false;
       for (var i = 0; i < numCompleteChunks; i++) {
         var message = JSON.parse(chunks[i]);
```

Comment:
It has to be parsed first. `chunks` here is just an array of (complete) JSON strings.